### PR TITLE
setup auto redirect to https://ide.unpar.ac.id ~~https://moodledemo.pascal.id~~

### PR DESCRIPTION
**Pull request by Bephometh**
Set Moodle mobiles app ID's and automatically redirect it to the desired site ~~(https://moodledemo.pascal.id)~~ (https://ide.unpar.ac.id).

### CODE REVIEW

File: config.xml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <?xml version='1.0' encoding='utf-8'?>
-<widget android-versionCode="39300" id="com.moodle.moodlemobile" ios-CFBundleVersion="3.9.3.0" version="3.9.3" versionCode="39300" xmlns="http://www.w3.org/ns/widgets" xmlns:android="http://schemas.android.com/apk/res/android" xmlns:cdv="http://cordova.apache.org/ns/1.0">
-    <name>Moodle</name>
-    <description>Moodle official app</description>
-    <author email="mobile@moodle.com" href="http://moodle.com">Moodle Mobile team</author>
+<widget android-versionCode="10000" id="com.ide.mobile" ios-CFBundleVersion="1.0.0.0" version="1.0.0" versionCode="10000" xmlns="http://www.w3.org/ns/widgets" xmlns:android="http://schemas.android.com/apk/res/android" xmlns:cdv="http://cordova.apache.org/ns/1.0">
```
Comment (pascalalfadian) [inline config.xml]: id nya gunakan `id.ac.unpar.moodlemobile`, disesuaikan dengan domain UNPAR.

File: config.xml [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+    <name>IDE UNPAR Mobile</name>
+    <description>IDE UNPAR app</description>
+    <author email="gabriel.lazuardi@gmail.com" >Gabriel Panji Lazuardi</author>
```
Comment (pascalalfadian) [inline config.xml]: Gunakan e-mail student

File: src/config.json [hunk 3]
```diff
@@ -1,9 +1,9 @@
 {
-    "app_id": "com.moodle.moodlemobile",
-    "appname": "Moodle Mobile",
-    "desktopappname": "Moodle Desktop",
-    "versioncode": 3930,
-    "versionname": "3.9.3",
+    "app_id": "com.ide.mobile",
```
Comment (pascalalfadian) [inline src/config.json]: Ini juga, disesuaikan

File: src/config.json [hunk 4]
```diff
@@ -63,25 +63,15 @@
...
-            "username": "teacher",
-            "password": "moodle"
-        }
-    },
     "font_sizes": [
         62.5,
         75.89,
         93.75
     ],
+    "presets" : {"url" :"https://moodledemo.pascal.id/"},
```
Comment (pascalalfadian) [inline src/config.json]: Catat URL forum yang memberitahu kamu bagaimana cara mengubah ini. Nanti di dokumen skripsi mesti ada juga.

**Review by pascalalfadian — CHANGES_REQUESTED** [review]

**Comment by Bephometh** [comment]
Link to forums that discussed the auto redirect setup : https://moodle.org/mod/forum/discuss.php?d=280042

**Comment by Bephometh** [comment]
Apparently, to deploy on mobile you need to change the app id in `google-service.json` since the default app id was chaged and conflicted with the app id in `google-service.json` this was not mentioned in their documentation

**Pull request merged**